docs/UserGuide.md

```diff
@@ -281,7 +271,7 @@ Format: `add n/LANDLORD_NAME p/PHONE_NUMBER l/LOCATION a/ASKING_PRICE t/PROPERTY
 Examples:
 * `add n/John p/87152433 l/Paya Lebar a/200,000 t/Condominium`
 
-### Listing all buyers : `view`
+### Viewing all properties : `view`
```

Comment:
Good catch on the language used, this makes it easier for new users or users that might not have the technical jargon to understand what the commands are used for.